### PR TITLE
Auto versioning

### PR DESCRIPTION
**Pull request by juliano**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,3 +1,7 @@
+plugins {
+    id 'net.researchgate.release' version '2.8.1'
+}
+
```
Comment (gustavomonarin) [inline build.gradle:4]: bellow here, on the line 13 we still have the property version, which is a different one from the properties file. Can we remove it?

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+version=0.1.1-SNAPSHOT
```
Comment (gustavomonarin) [inline gradle.properties:1]: What is the idea of this properties file? to be manually edited? How and when this is going to be changed?

**Review by gustavomonarin — COMMENTED** [review]
Should we also consider changing the .github workflow to introduce something like this: 
https://github.com/researchgate/gradle-release#working-in-continuous-integration

Or do u have any other idea of flow/process?

File: gradle.properties (hunk 2 above)
Comment (juliano) [reply gradle.properties:1]: The plugin keeps track of this file, that's how it knows the current/new version

**Comment by juliano** [comment]
Nope, no idea, I am not too familiar with gradle, so the changes are baby steps. I am not still in this part of the guide :D

**Pull request merged**